packages/rpc-spec/src/__tests__/rpc-shared-test.ts

```diff
@@ -0,0 +1,41 @@
+import '@solana/test-matchers/toBeFrozenObject';
+
+import { createJsonRpcResponseTransformer, RpcRequest, RpcResponse } from '../rpc-shared';
+
+describe('createJsonRpcResponseTransformer', () => {
+    it('can alter the value of the json Promise', async () => {
+        expect.assertions(2);
```

Comment:
Suspect that this should be two tests whenever you see this >1. ‘transforms the value of the `json` promise’ and ‘does not transform the value of the `text` promise.’